gateway/health_check.go

```diff
@@ -173,9 +175,7 @@ func gatherHealthChecks() {
 				Time:          time.Now().Format(time.RFC3339),
 			}
 
-			rpcStore := RPCStorageHandler{KeyPrefix: "livenesscheck-"}
-
-			if !rpcStore.Connect() {
+			if !rpc.GroupLogin() {
```

Comment:
We also have 'Login' function there. And we run either group login or standard login depending on the setup. 
Also, this logic is built in inside rpc Login function:
```
	// If we have a group ID, lets login as a group
	if config.GroupID != "" {
		return GroupLogin()
	}
```
So feels like replacing `GroupLogin` with `Login` should do the job.

Reply:
Change done and tested